.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  "stories": ["../src/**/*.stories.mdx", "../src/**/*.stories.@(ts|tsx)"],
```

Comment:
nit: do we need `.mdx` in the config?

Reply:
That depends on whether or not we do want to use mdx files to document.

Reply:
I think it is useful to be able to add some documentation / text to a story in a standardised manner, providing a way to share other resources, caveats, best practice, etc.